### PR TITLE
Ice clean up x2

### PR DESCRIPTION
**Pull request by mikepmanning**

### CODE REVIEW

**Comment by mikepmanning** [comment]
@keneanung - can you take a look at this - trying to stick in a new pull request to update some lines and ran into a check fail. I'm not sure what I'm looking at here.

Thanks

**Comment by keneanung** [comment]
I didn't at first either, but on second glance, I understand: the branch does not have the travis setting file (`.travis.yml`) yet, which is why the `push` hook fails. The `pr` hook thecks the result of a possible merge, where the file already exists.

You have 2 options: Merge the master branch into your `Ice-clean-up-x2` branch or ignore that fail as long as the `pr` is green.

File: raw-mm.defs.lua [hunk 1]
```diff
@@ -2655,6 +2659,8 @@ defences.complete_def({name = "garb", def = "You are wearing a garb of shadows."
 
 defences.complete_def({name = "Shadow Shroud", def = "You are wearing a magic shroud.", tooltip = "Conceals most actions."})
 
+defences.complete_def({name = "Mindfield Arti", def = "You will discharge a psychic lash from a powerful artifact on those who scry.", tooltop = "damages those who scry you"})
```
Comment (vadimnehta) [inline raw-mm.defs.lua]: tooltip, not tooltop

Comment (vadi2) [inline raw-mm.defs.lua]: tooltip, not tooltop

**Comment by vadi2** [comment]
Looks good, merge when you're ready.

**Comment by mikepmanning** [comment]
Ok, looks like I was able to merge and re-open and get the checks to work

Also fixed the typo

**Pull request merged**